From bd41dd93f37df79c7e14293c26dbff5abe5e4ce4 Mon Sep 17 00:00:00 2001
From: Vlad Zahorodnii <vlad.zahorodnii@kde.org>
Date: Tue, 21 Mar 2023 20:31:44 +0200
Subject: [PATCH] Drop unused kitemviews dependency

Seems unused. KWin compiles fine without it.

(cherry picked from commit 2c0acdca11dd25b0a3d5de8125c61d3523e83fbd)
---
 CMakeLists.txt                  | 1 -
 src/kcms/scripts/CMakeLists.txt | 1 -
 2 files changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f98937161f..d49f8a9471 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -90,7 +90,6 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
     NewStuff
     Service
     XmlGui
-    ItemViews
 )
 
 find_package(Threads)
diff --git a/src/kcms/scripts/CMakeLists.txt b/src/kcms/scripts/CMakeLists.txt
index 66966d6d46..fd7a368a97 100644
--- a/src/kcms/scripts/CMakeLists.txt
+++ b/src/kcms/scripts/CMakeLists.txt
@@ -18,7 +18,6 @@ target_link_libraries(kcm_kwin_scripts
     KF5::NewStuffWidgets
     KF5::Package
     KF5::QuickAddons
-    KF5::ItemViews
 )
 
 kpackage_install_package(package kcm_kwin_scripts kcms)
-- 
2.42.0

